draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -195,7 +187,7 @@ Included in the model are representations of configuration, states, and telemetr
 
```

Comment:
Isn't this draft about he gap analysis? I would suggest to split the current text into two drafts:
- gap analysis: Identifies gaps related to existing work in IETF
- pluggable modeling: adding YANG models to to cover pluggable specifics referring to the gaps identified earlier